schemas/logs/gravitational/teleport_audit.yml

```diff
@@ -172,14 +169,7 @@ fields:
       - ip
   - name: dst_port
     description: Destination port
-    type: object
```

Comment:
Column changes like this are not allowed - they require data migration.

Reply:
See https://github.com/panther-labs/panther-analysis/pull/229#discussion_r604046134